### PR TITLE
collector for queued RFCs added

### PR DESCRIPTION
**Pull request by JohannesKonings**
https://github.com/abap-observability-tools/abap-metrics-provider/issues/117

Documentation will follow :)

### CODE REVIEW

File: src/zamp_collector/zcl_amp_c_queued_rfcs.clas.abap [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    WHERE qrfcdatum = @date_current_run
+    GROUP BY qname, dest, qstate.
+
+    LOOP AT inbound_rfcs ASSIGNING FIELD-SYMBOL(<inbound_rfc>).
+
+      name = |{ <inbound_rfc>-qname }_{ <inbound_rfc>-dest }|.
+
+      status = <inbound_rfc>-status.
+
+      DATA(inbound_metric) = VALUE zif_amp_collector=>metric( metric_key = status metric_value = <inbound_rfc>-count ).
```
Comment (Goala) [inline src/zamp_collector/zcl_amp_c_queued_rfcs.clas.abap:63]: is intended that the metric_key only consits of the status? 
I think you forgot the add the name

File: src/zamp_collector/zcl_amp_c_queued_rfcs.clas.abap [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    WHERE qrfcdatum = @date_current_run
+    GROUP BY qname, dest, qstate.
+
+    LOOP AT outbound_rfcs ASSIGNING FIELD-SYMBOL(<outbound_rfc>).
+
+      name = |{ <outbound_rfc>-qname }_{ <outbound_rfc>-dest }|.
+
+      status = <outbound_rfc>-status.
+
+      DATA(outbound_metric) = VALUE zif_amp_collector=>metric( metric_key = status metric_value = <outbound_rfc>-count ).
```
Comment (Goala) [inline src/zamp_collector/zcl_amp_c_queued_rfcs.clas.abap:41]: is intended that the metric_key only consits of the status?
I think you forgot the add the name

**Review by Goala — CHANGES_REQUESTED** [review]

File: src/zamp_collector/zcl_amp_c_queued_rfcs.clas.abap (hunk 1 above)
Comment (JohannesKonings) [reply src/zamp_collector/zcl_amp_c_queued_rfcs.clas.abap]: your right. fixed it.

File: src/zamp_collector/zcl_amp_c_queued_rfcs.clas.abap (hunk 2 above)
Comment (JohannesKonings) [reply src/zamp_collector/zcl_amp_c_queued_rfcs.clas.abap]: your right. fixed it.

**Review by Goala — APPROVED** [review]
LGTM

**Pull request merged**